module/VuFind/tests/integration-tests/src/VuFindTest/Mink/HomePageFacetsTest.php

```diff
@@ -67,7 +67,7 @@ public function testHierarchicalFacets()
         $session->visit($this->getVuFindUrl() . '/Search/Home');
         $page = $session->getPage();
         $this->waitForPageLoad($page);
-        $container = $this->findCss($page, '#facet_hierarchical_facet_str_mv');
+        $container = $this->findCss($page, '.home-facet.hierarchical_facet_str_mv .home-facet-list');
```

Comment:
Given the bug that @sturkel89 found and @EreMaijala fixed, should we click on one of these and assert something about where we land?

Reply:
Good idea. I've expanded the test and added one for normal facets as well while at it.